app/controllers/datastreams_controller.rb

```diff
@@ -40,11 +40,11 @@ def update
       # if the content is not well-formed xml, inform the user rather than raising an exception
       error_msg = 'The datastream could not be saved due to malformed XML.'
     rescue Dor::Services::Client::UnexpectedResponse => e
-      error_msg = e.errors.first['detail']
+      error_msg = e.errors.first['detail']&.truncate(1024) # Truncate to avoid cookie overflow. Cookies can hold 4k.
```

Comment:
Should this be configuration?

Reply:
I find think that would just make it harder to track down.

Reply:
Yeah, but we're unlikely to remember that it needs to be changed in multiple locations. (Not a blocker, just a comment.)

Reply:
@justinlittman  it might vary by location because there is a different preamble in each place.